plugins/woocommerce/includes/rest-api/Controllers/Version4/Settings/class-wc-rest-general-settings-v4-controller.php

```diff
@@ -353,9 +351,6 @@ private function sanitize_setting_value( $setting_type, $value ) {
 				// If it's a string, convert to array (for single values).
 				return is_string( $value ) ? array( sanitize_text_field( $value ) ) : array();
 
-			case 'checkbox':
-				return wc_bool_to_string( $value );
-
```

Comment:
I see that the client now sends 'yes' and 'no' properly, but I think we should still sanitize on the API just to be safe.